qemu/tests/cfg/balloon_stress.cfg

```diff
@@ -1,26 +1,32 @@
 - balloon_stress:
     type = balloon_stress
     virt_test_type = qemu
-    only Windows
     kill_vm = yes
     repeat_times = 500
     balloon_timeout = 600
-    video_test = win_video_play
     balloon = balloon0
     balloon_dev_devid = balloon0
     balloon_dev_add_bus = yes
-    video_url = http://FILESHARE.COM/pub/section2/kvmauto/video/big_buck_bunny_480p_stereo.avi
-    x86_64:
-        program_files = "%ProgramFiles(x86)%"
-    i386:
-        program_files = "%ProgramFiles%"
-    driver_name = "balloon.sys"
-    #Disable first startup guide for windows media player
-    wmplayer_reg_cmd = "reg add HKLM\SOFTWARE\Policies\Microsoft\WindowsMediaPlayer /v GroupPrivacyAcceptance  /t REG_DWORD /f /d 00000001"
-    wmplayer_path = "${program_files}\Windows Media Player\wmplayer.exe"
-    #Install kmplayer if wmplayer is not installed default
-    kmplayer_install_cmd = "start /wait WIN_UTILS:\kmplayer\%s\KMPlayer-setup.exe /SP- /VERYSILENT"
-    kmplayer_path = "${program_files}\KMPlayer\kmplayer.exe"
-    play_video_cmd = '"%s" "%s" /play /fullscreen'
-    time_for_video = 1200
-    guest_alias = "Win2008-sp2-32:2k8\x86,Win2008-sp2-64:2k8\amd64,Win2008-r2-64:2k8\amd64,Win2012-64:2k12\amd64,Win2012-64r2:2k12\amd64"
+    Windows:
+        stress_test = win_video_play
+        video_url = http://FILESHARE.COM/pub/section2/kvmauto/video/big_buck_bunny_480p_stereo.avi
+        x86_64:
+            program_files = "%ProgramFiles(x86)%"
+        i386:
+            program_files = "%ProgramFiles%"
+        driver_name = "balloon.sys"
+        #Disable first startup guide for windows media player
+        wmplayer_reg_cmd = "reg add HKLM\SOFTWARE\Policies\Microsoft\WindowsMediaPlayer /v GroupPrivacyAcceptance  /t REG_DWORD /f /d 00000001"
+        wmplayer_path = "${program_files}\Windows Media Player\wmplayer.exe"
+        #Install kmplayer if wmplayer is not installed default
+        kmplayer_install_cmd = "start /wait WIN_UTILS:\kmplayer\%s\KMPlayer-setup.exe /SP- /VERYSILENT"
+        kmplayer_path = "${program_files}\KMPlayer\kmplayer.exe"
+        play_video_cmd = '"%s" "%s" /play /fullscreen'
+        time_for_video = 1200
+        guest_alias = "Win2008-sp2-32:2k8\x86,Win2008-sp2-64:2k8\amd64,Win2008-r2-64:2k8\amd64,Win2012-64:2k12\amd64,Win2012-64r2:2k12\amd64"
+    Linux:
+        stress_test = autotest_control
+        test_control_file = stress.control
```

Comment:
With latest strategy,  it is not recommended to use autotest related module, case and so no.  
Try to find another good method to do this.